### ISSUE TITLE
Node.js Foundation Security WorkGroup Meeting 2018-11-01

### ISSUE THREAD

**Issue by mhdawson** [nodejs/security-wg#426]
## Time

**UTC Thu 01-Nov-2018 15:00 (03:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Thu 01-Nov-2018 8:00 (8:00 AM) |
| US / Mountain | Thu 01-Nov-2018 9:00 (9:00 AM) |
| US / Central  | Thu 01-Nov-2018 10:00 (10:00 AM) |
| US / Eastern  | Thu 01-Nov-2018 11:00 (11:00 AM) |
| London        | Thu 01-Nov-2018 15:00 (03:00 PM) |
| Amsterdam     | Thu 01-Nov-2018 16:00 (04:00 PM) |
| Moscow        | Thu 01-Nov-2018 18:00 (6:00 PM) |
| Chennai       | Fri 02-Nov-2018 20:30 (8:30 PM) |
| Hangzhou      | Fri 02-Nov-2018 23:00 (11:00 PM) |
| Tokyo         | Fri 02-Nov-2018 00:00 (00:00 AM) |
| Sydney        | Fri 02-Nov-2018 02:00 (02:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Security%20WorkGroup+Meeting+2018-11-01&iso=20181101T15
* or http://www.wolframalpha.com/input/?i=03PM+UTC%2C+Nov+01%2C+2018+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/10AqdOQufl5n4V7_DlXkkCe-GQZiKCInKP6rxD770pMo/edit>

## Agenda

Extracted from **security-wg-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/security-wg

* What is the good time for the Security WG monthly meeting [#416](https://github.com/nodejs/security-wg/issues/416)
* \[ANN\]: refactor report id field to be auto-generating, Deadline: 10.11.2018 [#402](https://github.com/nodejs/security-wg/issues/402)
* Migrate vulnerability database to it own repo [#359](https://github.com/nodejs/security-wg/issues/359)
* Create issue to propose session at JSInteractive collaborator summit [#358](https://github.com/nodejs/security-wg/issues/358)

## Invited

* Security wg team: @nodejs/security-wg

## Notes

The agenda comes from issues labelled with `security-wg-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

* link for participants: https://zoom.us/j/405474359
* For those who just want to watch We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at https://www.youtube.com/c/nodejs+foundation/live when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up. 
* youtube admin page: https://www.youtube.com/my_live_events?filter=scheduled

**Comment by sam-github** [comment]
@mhdawson we aren't going to use the new meeting time from #416 ?

**Comment by mhdawson** [comment]
Meeting is auto-generated from what is in the calendar :)

Now I was just about to say that I can't make the existing time this week so I'm going to update to the new time.

**Comment by mhdawson** [comment]
Updated times in invite, hope I got them right.
@nodejs/security-wg FYI

**Comment by mhdawson** [comment]
I will have to leave early after the first 35 mins, but I think I can leave the meeting without dropping everybody else.

**Comment by sam-github** [comment]
@mhdawson You said you updated something, but the invites here and in the node foundation calendar are still for Nov 1.

**Comment by Elexy** [comment]
I've had a maintainer ask for guidance on how to handle publicity around an issue and subsequent patched release. Do we have any protocol? and if not could we maybe write something up in this repo to point them to?

**Comment by MarcinHoppe** [comment]
@Elexy Maybe this is something to add to the triage guidelines doc? I think all the comments in the Google Doc have been addressed and I want to move it to this repo where we can work on it in a PR.

**Comment by mhdawson** [comment]
Going to close this as the meeting was rescheduled.  @Elexy I'd suggest opening a new issue to continue your discussion.
